src/m365/yammer/commands/group/group-user-remove.ts

```diff
@@ -113,8 +89,32 @@ class YammerGroupUserRemoveCommand extends YammerCommand {
       });
 
       if (result.continue) {
-        await executeRemoveAction();
+        await this.executeRemoveAction(args);
+      }
+    }
+  }
+
+  private async executeRemoveAction(args: CommandArgs): Promise<void> {
+    const endpoint = `${this.resource}/v1/group_memberships.json`;
```

Comment:
I guess we don't need this variable. Lets move this URL directly to the `resuestOptions`